spec/fixtures/cassettes/create_user.yml

```diff
@@ -0,0 +1,51 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://harvest.greenhouse.io/v1/users
+    body:
+      encoding: UTF-8
+      string: '{"first_name":"Pika","last_name":"Pikachu","email":"pikachu@testing.com","employee_id":"30928902"}'
+    headers:
+      Authorization:
+      - Basic MDcyOTVlM2YxNmYzNDI0MzFhNTlmMTcyM2M3MmRkMTUtMzo=
```

Comment:
i think this is a valid api key -- should we revoke and change this and the other cassettes?

Reply:
oooo I see, I'll change it to the same one from the previous cassettes then - thanks for catching!

Reply:
nice! i think we can revoke too in case someone on github sees this commit history :D